src/Service/BuildPackageService.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Service\Transifex\Connector\Languages;
+use App\Service\Transifex\DTO\PackageDTO;
+use Mautic\Transifex\Exception\ResponseException;
+use Mautic\Transifex\TransifexInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class BuildPackageService
+{
+    public function __construct(private readonly TransifexInterface $transifex, private readonly Filesystem $filesystem)
+    {
+    }
+
+    public function build(PackageDTO $packageDTO, ConsoleLogger $logger): int
```

Comment:
```suggestion
    public function build(PackageDTO $packageDTO, \Psr\Log\LoggerInterface $logger): int
```
The point of using the `$logger` instead of the `$io` was to completely decouple the services from the command. In general please prefer using interfaces to implementation (classes).